hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/result/SubscriptionChangeListener.java

```diff
@@ -0,0 +1,7 @@
+package pl.allegro.tech.hermes.consumers.consumer.result;
+
+import pl.allegro.tech.hermes.api.Subscription;
+
+public interface SubscriptionChangeListener {
```

Comment:
This name is a little bit unclear in the context of handler. What about `SubscriptionChangeAwareSuccessHandler`?
```
public interface SubscriptionChangeAwareSuccessHandler extends SuccessHandler {
  void updateSubscription(Subscription subscription);
}
```
It would be consistent with other handlers which need to add some additional features, just like `ConsumerAuthorizationHandler`